src/otx/algorithms/detection/configs/instance_segmentation/resnet50_maskrcnn/template.yaml

```diff
@@ -29,7 +29,7 @@ hyper_parameters:
         default_value: 4
         auto_hpo_state: POSSIBLE
       learning_rate:
-        default_value: 0.001
+        default_value: 0.007
```

Comment:
Looks OK, but please be careful when changing the default recipe params. Actually, we cannot be sure if it will be also better on tiling scenario, but suppose it'll be fair enough. @eugene123tw Any comments?

Reply:
Seems like supervised training also will benefit from it (I gave experiment results above). I haven't checked tiling though